src/bot/messaging.rs

```diff
@@ -61,6 +71,71 @@ impl Bot {
         self.post_message(&path, &params)
     }
 
+    /// API call whick will send the requested chat action to your bot.
+    pub fn send_chat_action(&self, chat_id: &i64, action: &ChatAction) -> Result<bool> {
+        let action = get_chat_action(action);
+        println!("{:?}", &action);
+        let path = ["sendChatAction"];
+        let url = ::construct_api_url(&self.bot_url, &path);
+        let params = [("chat_id", chat_id.to_string()),
+            ("action", action)];
+        let mut data = self.client.post(&url).form(&params).send()?;
+        println!("{:?}", &data);
+
+        let rjson: Value = check_for_error(data.json()?)?;
+        let message_json = rjson.get("result").ok_or(JsonNotFound)?;
```

Comment:
I don't think `message_json` is a semantically correct name for this variable binding. This action will not return `Message` object in JSON form.